### DISCUSSION TITLE
Possible to skip "Compiling shader variants" ?

### DISCUSSION THREAD

**Discussion by mathisevaveo** [superunitybuild/buildtool#149] [Q&A] [answered]
Hello ! 

I discovered SuperUnityBuild very recently and find the package incredible! It's super practical for my corss-platform project and it's already saved me a huge amount of time! 

Unfortunately, Unity is still causing problems ahah! I wanted to know if with the package you could deactivate the "Compiling Shader Variants" because it takes almost 45 minutes per version even though I'm using a super M2! 

Thanks in advance for your answers! 
Best regards

**Comment by robinnorth** [comment, marked as answer]
Thanks for your kind words, @mathisevaveo!

Once you've created a build with shaders compiled, you can set your Release Type to only compile script changes for subsequent runs by selecting 'Build Scripts Only' option from the 'Advanced Options' dropdown:
<img width="364" alt="image" src="https://github.com/superunitybuild/buildtool/assets/3787255/94458106-2a01-4a65-b56e-4c2325e99bab">

Alternatively, I can highly recommend [Shader Control](https://assetstore.unity.com/packages/vfx/shaders/shader-control-74817) on the Unity Asset Store as a means of cutting down the number of shader variants that need to be compiled for your project.

Hope this helps!

**Comment by mathisevaveo** [comment]
Perfect, thank you, i will check this today !
